osbs/build/build_request.py

```diff
@@ -166,6 +166,8 @@ def dj(self):
     def has_ist_trigger(self):
         """Return True if this BuildConfig has ImageStreamTag trigger."""
         triggers = self.template['spec'].get('triggers', [])
+        if not triggers:
```

Comment:
To address the code coverage decrease, we could remove this if-block and change the for loop to:
```python
for trigger in triggers or []:
```